source/Mlos.Agent.Server/Mlos.Agent.Server.csproj

```diff
@@ -25,8 +25,19 @@
     <ProjectReference Include="$(SourceDir)\Mlos.NetCore\Mlos.NetCore.csproj" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="CliOptionsParser.cs" />
     <Compile Include="MlosAgentServer.cs" />
     <Compile Include="TargetProcessManager.cs" />
   </ItemGroup>
   <Import Project="$(BaseDir)\build\Mlos.NetCore.targets" />
+  <Target Name="TestBadCliArgs" AfterTargets="Build" Condition="'$(RunUnitTest)' != 'false'">
```

Comment:
Can you remove it ?

Reply:
Reverted in 57a091412a9aece9e08243772ade90add4302003